ecs.php

```diff
@@ -27,7 +35,6 @@
 
     //Import
     $services->set(\PhpCsFixer\Fixer\Import\NoLeadingImportSlashFixer::class);
-    $services->set(\PhpCsFixer\Fixer\Import\NoUnusedImportsFixer::class);
```

Comment:
Panie. Jak będziesz tak robił w PRach to będę je zamykał XD

Reply:
Myslalem, ze sie tam pomyliles czy cosXD przywroce zaraz. Przepraszam 😒

Reply:
Jestesmy tylko ludzmi. A no spoko to sie zrobi PR'ka.